introduction.md

```diff
@@ -0,0 +1,7 @@
+Talk about myself
```

Comment:
Hi, good afternoon.

I added the change requested.

Thanks for your attention.

Hugs.

Kelvin Matheus